ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTest.scala

```diff
@@ -101,7 +101,8 @@ abstract class HttpServiceIntegrationTest
       if (System.getProperty("os.name") == "Linux") label in fn else ()
   }
 
-  "should serve HTTPS requests" ifLinux withHttpService(useHttps = UseHttps.Https) { fixture =>
```

Comment:
I think this will complain about the unused `ifLinux`, I think we should comment it out for now

Reply:
good catch, let me fix that right away, on both PRs